apps/api/src/server/repositories/folder.repository.js

```diff
@@ -21,7 +21,7 @@ export const getByCaseId = (caseId, parentFolderId = null) => {
 /**
  * Returns a single folder on a case
  *
- * @param {number} folderId
+ * @param {number | any} folderId
```

Comment:
We should not allow `any` here

Reply:
what would be the alternative - can I let 'undefined'?

Reply:
It gives the red squiggly line if I don't 🙈

Reply:
If a TSCheck error appears, then we need to understand why and fix the source of the issue

Reply:
It says something along the lines of cannot use type 'number' on 'undefined'

Reply:
i removed it - the tschecks are passing - but the squiggly lines are still there